### ISSUE TITLE
Composed Physics- and Data-driven System Identification for Non-autonomous Systems in Control Engineering. (arXiv:2112.08148v1 [math.OC])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#14136]
Composed Physics- and Data-driven System Identification for Non-autonomous Systems in Control Engineering. (arXiv:2112.08148v1 [math.OC])<br>
https://ift.tt/3DZtDbP<br>
<p>In control design most control strategies are model-based and require accurate models to be applied successfully. Due to simplifications and the model-reality-gap physics-derived models frequently exhibit deviations from real-world-systems. Likewise, purely data-driven methods often do not generalise well enough and may violate physical laws. Recently Physics-Guided Neural Networks (PGNN) and physics-inspired loss functions separately have shown promising results to conquer these drawbacks. In this contribution we extend existing methods towards the identification of non-autonomous systems and propose a combined approach PGNN-L, which uses a PGNN and a physics-inspired loss term (-L) to successfully identify the system's dynamics, while maintaining the consistency with physical laws. The proposed method is demonstrated on two real-world nonlinear systems and outperforms existing techniques regarding complexity and reliability.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
